app/src/ui/preferences/appearance.tsx

```diff
@@ -27,66 +33,36 @@ const themes: ReadonlyArray<ISegmentedItem<ApplicationTheme>> = [
     description: 'GitHub Desktop is for you too, creatures of the night',
     key: ApplicationTheme.Dark,
   },
+  ...(supportsSystemThemeChanges() ? [systemTheme] : []),
 ]
 
 export class Appearance extends React.Component<IAppearanceProps, {}> {
-  private onSelectedThemeChanged = (value: ApplicationTheme) => {
-    this.props.onSelectedThemeChanged(value)
-    this.props.onAutomaticallySwitchThemeChanged(false)
+  private onSelectedThemeChanged = (theme: ApplicationTheme) => {
+    this.props.onSelectedThemeChanged(theme)
   }
 
-  private onAutomaticallySwitchThemeChanged = (
-    event: React.FormEvent<HTMLInputElement>
-  ) => {
-    const value = event.currentTarget.checked
+  public render() {
+    let selectedTheme = this.props.selectedTheme
 
-    if (value) {
-      this.onSelectedThemeChanged(
-        isDarkModeEnabled() ? ApplicationTheme.Dark : ApplicationTheme.Light
-      )
+    if (
+      this.props.selectedTheme === ApplicationTheme.System &&
+      !supportsSystemThemeChanges()
+    ) {
+      selectedTheme = isDarkModeEnabled()
+        ? ApplicationTheme.Dark
+        : ApplicationTheme.Light
     }
```

Comment:
If `supportsSystemThemeChanges` is `false`, then `isDarkModeEnabled` would never be true, right?
```suggestion
    if (
      this.props.selectedTheme === ApplicationTheme.System &&
      !supportsSystemThemeChanges()
    ) {
      selectedTheme = ApplicationTheme.Light
    }
```

Reply:
My understanding is no, even though your system can not emit events for Electron to pick up and automatically change with your system, you are still able to switch from DarkMode to light mode. The reason for it is the default setting is System now. Users on older systems will need this case.

Reply:
Edit: Ahh, I see what you mean, I think I didn't test that case (my OS is too new) but effectively we need to just return `remote.nativeTheme.shouldUseDarkColors` no matter what now.